rust-stage0.yaml

```diff
@@ -1,14 +1,14 @@
 package:
   name: rust-stage0
-  version: 1.65.0
-  epoch: 2
+  version: 1.68.2
```

Comment:
I'm surprised we're updating the stage0 package. Is that because 1.68 is needed to be able to build 1.69? If so, can we document that in rust.yaml somewhere?

Reply:
Yes, 1.68 is needed to build 1.69.